src/components/Learn/learn.css

```diff
@@ -0,0 +1,271 @@
+.learn {
+    position: relative;
+    overflow: hidden;
+}
+
+.jalansetapak {
+    position: absolute;
+    z-index: -1;
+    top: 3rem;
+    object-fit: cover;
+}
+
+.learn__start {
+    display: flex;
+    flex-direction: column;
+} 
+
+.learn__start h3 {
+    margin-bottom: 3rem;
+}
+
+.learn__start-for {
+    display: flex;
+    flex-direction: row;
+    gap: 3rem;
+}
+
+.for-content {
+    width: 40%;
+    text-align: justify;
+    border-radius: var(--border-radius);
+    box-shadow: 0 0 10px 5px var(--white);
+    backdrop-filter: blur(20px); 
+    padding: 1.5rem 2rem;
+}
+
+.for-content h3 {
+    margin-bottom: 0.3rem;
+}
+
+
+.learn__otherSources {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 2rem;
+    background-color: var(--gray);
+    mask: linear-gradient(transparent 0%, var(--gray) 8%, var(--gray) 92%, transparent 100%);
+}
+
+
+
+.otherSources-text {
+    flex: 1;
+}
+
+
+.learn__oj {
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+}
+
+.learn__oj-main,
+.learn__oj-notes {
+    background-color: var(--white);
+    border-radius: var(--border-radius);
+    color: var(--dark);
+    padding: 2rem 3rem;
+}
+
+.learn__oj-main {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.learn__oj-main h1 {
+    flex: 1.5;
+    margin-right: 1rem;
+}
+
+.learn__oj-main-text {
+    flex: 3;
+    display: flex;
+    flex-direction: column;
+}
+
+.oj-recommendations {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 2rem;
+    margin-top: 1rem;
+}
+
+.oj__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.oj__container img {
+    max-width: 4.2rem;
+    margin-bottom: 0.8rem;
+}
+
+.oj__container span {
+    color: var(--dark);
+}
+
+.learn__oj-notes {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.learn__oj-notes h2 {
+    margin-bottom: 0.5rem;
+}
+
+.learn__important {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 4.5rem;
+    backdrop-filter: blur(10px); 
```

Comment:
Optional, but you can add a color in the background to make the text color not conflict with the original color image. The background color will still be affected by the transparency and blur.

<details>
<summary>Example Result</summary>

![image](https://github.com/user-attachments/assets/e38c64d0-00ce-4e15-a420-b58e819a35f7)
</details>

```suggestion
    background-color: rgb(0 0 0 / 30%);
    backdrop-filter: blur(10px); 
```